apps/parsley/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "baseUrl": "src",
     "paths": {
       "@evg-ui/lib/*": ["../../../packages/lib/src/*"]
-    }
+    },
+    "module": "ESNext"
```

Comment:
Should we just be replacing the `module` field in the root `tsconfig.base.json`? I don't know that any package needs CommonJS (or if they do, they could override the base default)

Reply:
this opened a whole new can of worms so i'm sorry about the additional file changes! Also I just have to call out that `ts-node` legit just doesn't work with Node 20 and ESM modules period ([GitHub issue here](https://www.github.com/TypeStrong/ts-node/issues/1997)) so if you see funky workarounds... that's why....